tests/integration/components/cfb-form-editor/question-test.js

```diff
@@ -68,6 +68,35 @@ module("Integration | Component | cfb-form-editor/question", function(hooks) {
     assert.verifySteps(["after-submit"]);
   });
 
+  test("it respects existing metadata", async function(assert) {
+    assert.expect(4);
+
+    this.server.create("question", {
+      label: "Test Label",
+      slug: "test-slug",
+      type: "TEXT",
+      meta: { someMetaKey: "foobar" }
+    });
+
+    this.server.logging = true;
```

Comment:
This should be removed

Reply:
Resolved in b7fdd7af196999e23df3812ca3a57d263b94a727